builder/amazon/common/step_key_pair.go

```diff
@@ -53,10 +55,19 @@ func (s *StepKeyPair) Run(ctx context.Context, state multistep.StateBag) multist
 	}
 
 	ec2conn := state.Get("ec2").(*ec2.EC2)
+	var keyResp *ec2.CreateKeyPairOutput
 
 	ui.Say(fmt.Sprintf("Creating temporary keypair: %s", s.Comm.SSHTemporaryKeyPairName))
-	keyResp, err := ec2conn.CreateKeyPair(&ec2.CreateKeyPairInput{
-		KeyName: &s.Comm.SSHTemporaryKeyPairName})
+	err := retry.Config{
+		Tries:      11,
+		RetryDelay: (&retry.Backoff{InitialBackoff: 200 * time.Millisecond, MaxBackoff: 30, Multiplier: 2}).Linear,
```

Comment:
Hello there, thanks for opening.
 watch out, here `MaxBackoff` is the maximum duration to wait between failures, which is also an int, but `30` here means 30 nanosecond. I think you meant seconds.

```suggestion
		RetryDelay: (&retry.Backoff{InitialBackoff: 200 * time.Millisecond, MaxBackoff: 30*time.Second, Multiplier: 2}).Linear,
```

Reply:
Is that correct ? If so please accept my suggestion.

I will open a PR to document those fields a bit better

Reply:
doc here: #7737

Reply:
Yes, I think you are right. It should be `30 * time.Seconds`. I actually referred [create_tags](https://github.com/hashicorp/packer/blob/master/builder/amazon/common/step_create_tags.go#L93-L124) step which doesn't use `time.Seconds`. I think we should correct that too.

Reply:
Welps thanks for saying that's not good, I'll open a PR fix those :D

Reply:
If you want, I can do that for you! 🙂

Reply:
Ah ! too late I already added it to #7737 thanks though 🙂 that would have been nice